crates/subspace-farmer/src/legacy_multi_plots_farm.rs

```diff
@@ -166,6 +166,10 @@ impl LegacyMultiPlotsFarm {
         })
     }
 
+    pub fn single_plot_farms(&self) -> &'_ [SinglePlotFarm] {
```

Comment:
Do we need `'_` here? I thought lifetime here is the same as in `&self`?

Reply:
Actually yes, but there are some cases where with `rust_2018_idioms` lint (which we didn't enable here) will ask for `'_` in some cases.